swift/sajson/sajson-ffi.cpp

```diff
@@ -59,3 +60,33 @@ const unsigned char* sajson_get_input(sajson_document* doc) {
 size_t sajson_get_input_length(struct sajson_document* doc) {
     return unwrap(doc)->_internal_get_input().length();
 }
+
+// MARK: -
+/// TODO: The following should be removed when they can be ported to Swift:
+
+static sajson_value* wrap(sajson::value* doc) {
+    return static_cast<sajson_value*>(doc);
+}
+
+static sajson::value* unwrap(sajson_value* value) {
+    return static_cast<sajson::value*>(value);
+}
+
+const sajson_element* sajson_get_value_payload(struct sajson_value* value) {
+    return unwrap(value)->_internal_get_payload();
+}
+
+uint8_t sajson_get_value_type(sajson_value* value) {
+    return unwrap(value)->get_type();
+}
+
+sajson_value* sajson_create_value(size_t type, const size_t* payload, const unsigned char* input) {
+    auto inputCasted = reinterpret_cast<const char*>(input);
+    auto value = sajson::value(sajson::type(type), payload, inputCasted);
+    return wrap(new(std::nothrow) sajson::value(std::move(value)));
```

Comment:
(also you need to make sure you have a matching delete operator somewhere.  better to just do all of this in Swift itself)